cli/tests/pcluster/validators/test_cluster_validators.py

```diff
@@ -187,6 +188,45 @@ def test_cluster_name_validator_slurm_accounting(cluster_name, scheduling, shoul
     assert_failure_messages(actual_failures, expected_message)
 
 
+@pytest.mark.parametrize(
```

Comment:
I'm not seeing a unit test on the generic warning. Could we add one?

Reply:
It is a different validator ..not sure what to test about it apart from its presence/absence.
I'll try to figure out a way to test it.

Reply:
I added a test for the Warning